crawler/src/main/java/com/brandwatch/minibcr/crawler/crawlerjob/Crawler.java

```diff
@@ -0,0 +1,21 @@
+package com.brandwatch.minibcr.crawler.crawlerjob;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import com.brandwatch.minibcr.common.domain.Resource;
+import com.brandwatch.minibcr.crawler.services.CrawlerService;
+
+@Component
+public class Crawler {
+
+    @Autowired
+    CrawlerService crawlerService;
```

Comment:
Access modifier?

Reply:
Also try not to use field injection, constructor injection is preferred. I think there should be an intellij suggestion for this.

Reply:
For some reason I assumed field injection was the recommended approach as it is one of the main "perks" of Springboot. Is there a particular reason why it should be avoided?

Reply:
In general it is recommended to avoid using filed injection. Maybe this [thread](https://stackoverflow.com/questions/39890849/what-exactly-is-field-injection-and-how-to-avoid-it/39892204#39892204) will explain more.

Reply:
This will probably explain it better than I can, [Field injection considered harmful](https://www.vojtechruzicka.com/field-dependency-injection-considered-harmful/)

Reply:
I'm convinced.